test/dotnet/scenarios.json

```diff
@@ -16,11 +16,60 @@
             }
         }
     },
-    "install_dotnet_5": {
+    "install_dotnet_6_bullseye": {
+        "image": "debian:bullseye",
+        "features": {
+            "dotnet": {
+                "version": "6"
+            }
+        }
+    },
+    "install_dotnet_6_focal": {
         "image": "ubuntu:focal",
         "features": {
             "dotnet": {
-                "version": "5"
+                "version": "6"
+            }
+        }
+    },
+    "install_dotnet_7_bullseye": {
```

Comment:
:info I'm sure you must be familiar, however fyi, the [test.sh](https://github.com/devcontainers/features/blob/01bbe2445f73c741aa6f0b05d93f5ecd259fab0f/test/dotnet/test.sh) file is executed for the Dotnet Feature (with default values (`7` in this case)) against these base images https://github.com/devcontainers/features/blob/main/.github/workflows/test-all.yaml#L45-L51

These dotnet scenarios for `7` are also covered under those, but I agree it's great to have them as the default could change anytime.

Reply:
Yep that was the idea!